tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26,py27,py34,py35,py36,py37,flake8
+envlist = py26,py27,py34,py35,py36,py37,py313,flake8
```

Let's remove the old ones and add the intermediary ones?

```suggestion
envlist = py34,py35,py36,py37,py38,py39,py310,py311,py312,py313,flake8
```